hooks/addon

```diff
@@ -120,7 +120,7 @@ upload_stemcells() {
     shift
   done
 
-  if [[ ${#versions[@]} -gt 0 ]] ; then
+  if [[ ${versions[@]} -gt 0 ]] ; then
```

Comment:
The "${#variable[@]}" returns the number of elements in the array.  Why did this need to be changed?

Reply:
Sorry, my fix was wrong. 

The latest commit fixed it now.
when versions is declared but not set, `${#versions[@]}` triggers an unbound variable when I upload the first stemcell. I tested that setting empty array as initialized value for versions solved issue without breaking the next stemcell upload.

Reply:
I was always curious if `declare -a lst` was enough for `-u`; apparently on older bashes (looking at you, 3.x) it was, but on newer 4.x lines it's not.

Good to know that "works on my machine" still isn't a great engineering discipline 😉